spring-boot/admin/src/main/resources/templates/admin/fragments/wizard/wizard.html

```diff
@@ -1,348 +1,389 @@
+            $("#serviceForFree").val(false);
+            // This hides the message, in case the user first clicked yes and then clicked no
+            $("#clickNextButtonMessage").hide();
+        });
+
+        $('#monthlyCharge').click(function (evt) {
+            $("#monthlyCharge").attr('class', 'btn btn-danger');
+            $("#percallCharge").attr('class', 'btn btn-primary');
+            $("#monthlyChargeInput").val(true);
+            $("#percallChargeInput").val(false);
+            $("#amountDiv").attr('style', 'display: block');
+        });
+
+        $('#percallCharge').click(function (evt) {
+            $("#percallCharge").attr('class', 'btn btn-danger');
+            $("#monthlyCharge").attr('class', 'btn btn-primary');
+            $("#monthlyChargeInput").val(false);
+            $("#percallChargeInput").val(true);
+            $("#amountDiv").attr('style', 'display: block');
+        });
+        // This function will check if the user inserted an input in the service value
+        $("#serviceCost").change(function () {
+            $("#serviceCost").val() == '' ? $("#clickNextButtonMessage").hide() : $("#clickNextButtonMessage").show();
+        });
+    });
+
+    var smartWizard = null;
+
+    function init_SmartWizard() {
+
+        if (typeof ($.fn.smartWizard) === 'undefined') {
+            return;
+        }
+
+        smartWizard = $('#wizard').smartWizard(
             {
-                    $("#serviceNameSummary").html($("#serviceName").val());
-                    $("#serviceUrlSummary").html($("#serviceUrl").val());
-
-                    if ($("#monthlyChargeInput").val() == "true")
-                    {
-                        $("#costMethod").html("Monthly charge")
-                    }
-
-                    if ($("#monthlyChargeInput").val() == "true")
-                    {
-                        $("#costMethod").html("Pay per month, " + $("#serviceCost").val() + " € per month")
-                    }
-
-                    if ($("#percallChargeInput").val() == "true")
-                    {
-                        $("#costMethod").html("Pay per number of calls, " + $("#serviceCost").val() + " € per 1000 calls")
-                    }
-
-                    if ($("#serviceForFree").val() == "true")
-                    {
-                        $("#costMethod").html("Service For Free")
-                    }
+                autoAdjustHeight: true,
+                keyNavigation: false,
+                onLeaveStep: leaveAStepCallback,
+                onShowStep: onShowStepEvent,
+                onFinish: onFinishCallback
+            }
+        );
+
+        $('.buttonNext').addClass('btn btn-success');
+        $('.buttonPrevious').addClass('btn btn-primary');
+        $('.buttonFinish').addClass('btn btn-default');
+
+    }
+
+    function leaveAStepCallback(obj, context) {
+        if (context.toStep > context.fromStep)
+            return validateSteps(context.fromStep);
```

Comment:
`validateSteps` is a little bit confusing because actually the user input is validated, so maybe rename it as
```suggestion
            return validateUserInputForStep(context.fromStep);
```